src/System.IO.FileSystem/tests/File/ReadWriteAllTextAsync.cs

```diff
@@ -61,10 +62,13 @@ public class File_ReadWriteAllTextAsync : FileSystemTest
         [Theory]
         [InlineData(0)]
         [InlineData(100)]
+        [InlineData(4096)]
```

Comment:
100 seemed to cover it as a value less than the built-in buffer size, combined with changing the input to actually verify the right sequence of values.